benchexec/tools/fshell-witness2test.py

```diff
@@ -62,7 +66,10 @@ def cmdline(self, executable, options, task, rlimits):
         if data_model_param and data_model_param not in options:
             options += [data_model_param]
 
-        return [executable] + options + list(task.input_files_or_identifier)
+        input_files = get_non_witness_input_files(task)
+        witness_options = ["--graphml-witness"]
```

Comment:
Hm, this seems to hard-code an assumption about what specific format the witness is in. But of course this is not guaranteed. What to do here?

Reply:
The option was taken from the benchmark definition file for `fshell-witness2test`. Therefore I think this is tool dependent on how to handle this. Currently they also only accept witnesses in version 1.0 for which the option makes sense.